src/components/footer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { TodoCount } from './todoCount';
+import { Todo } from '../types/Todo';
+import { FilterOption } from '../types/filterOption';
+
+interface Props {
+  handleSetFilter: (newFilter: FilterOption) => void;
+  todos: Todo[];
+  setTodos: (todos: Todo[]) => void;
+}
+
+// eslint-disable-next-line max-len
+export const Footer: React.FC<Props> = ({ handleSetFilter, todos, setTodos }) => {
```

Comment:
```suggestion
export const Footer: React.FC<Props> = ({
handleSetFilter,
todos,
setTodos }) => {
```
I know it's looks worst but then you don't have to disabling eslint